src/utils/Params.h

```diff
@@ -49,4 +49,18 @@ class Params {
                                     BoundaryType::SOFT_REFLECTIVE,
                                     BoundaryType::SOFT_REFLECTIVE,
                                     BoundaryType::SOFT_REFLECTIVE };
+    // initial temperature
+    double init_temp = 0;
+    // target temperature
+    double target_temp = 0;
+    // frequency of thermostat updates
+    unsigned thermo_freq = 0;
+    // maximum temperature delta
+    double max_temp_delta = 0.01;
+    // gravitational constant
+    double gravity = 0.0;
+    // particle types 
+    std::vector<std::pair<double, double>> particleTypes;
```

Comment:
An sich brauche ich zumindest den Vektor so in der Form nicht. Wenn wir den sonst nicht brauchen, würde ich ihn einfach weglassen